sentry/src/main/java/io/sentry/IScopeObserver.java

```diff
@@ -1,18 +1,20 @@
 package io.sentry;
 
 import io.sentry.protocol.User;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 /** Observer for the sync. of Scopes across SDKs */
 public interface IScopeObserver {
-  void setUser(User user);
+  void setUser(@Nullable User user);
 
-  void addBreadcrumb(Breadcrumb crumb);
+  void addBreadcrumb(@NotNull Breadcrumb crumb);
 
-  void setTag(String key, String value);
+  void setTag(@NotNull String key, @Nullable String value);
```

Comment:
I believe Sentry is unhappy with a `null` value and we have an API to `removeTag` so here we're probably better off with `NotNull`

Reply:
Good spot. This was my mistake.